lattices/src/ord.rs

```diff
@@ -41,18 +41,9 @@ impl<T> ConvertFrom<Max<T>> for Max<T> {
     }
 }
 
-impl<T> Compare<Max<T>> for Max<T>
-where
-    T: Ord,
-{
-    fn compare(&self, other: &Max<T>) -> Option<std::cmp::Ordering> {
-        Some(Ord::cmp(&self.0, &other.0))
-    }
-}
-
 /// A totally ordered min lattice. Merging takes the smaller value.
 #[repr(transparent)]
-#[derive(Default, PartialEq, PartialOrd, Eq, Ord, Debug, Clone)]
+#[derive(Copy, Clone, Debug, Default, PartialOrd, Ord, PartialEq, Eq)]
```

Comment:
!!! This `PartialOrd, Ord` will be backwards. In the min lattice, 0 > 1

Could also maybe try `type Min<T> = Max<std:::cmp::Reverse<T>>` or something, not sure how ergonomic that is

Reply:
nice catch :)

Reply:
yeah, wrapping the reverse in there makes it pretty unergonomic because now you can't do Min::new(0);

Reply:
Its a shame